flaps/flaps_secrets.go

```diff
@@ -0,0 +1,46 @@
+package flaps
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	fly "github.com/superfly/fly-go"
+)
+
+func (f *Client) sendRequestSecrets(ctx context.Context, method, endpoint string, in, out interface{}, headers map[string][]string) error {
+	endpoint = fmt.Sprintf("/apps/%s/secrets%s", f.appName, endpoint)
+	return f._sendRequest(ctx, method, endpoint, in, out, headers)
+}
+
+func (f *Client) ListSecrets(ctx context.Context) ([]fly.ListSecret, error) {
+	ctx = contextWithAction(ctx, secretsList)
+
+	out := make([]fly.ListSecret, 0)
+	if err := f.sendRequestSecrets(ctx, http.MethodGet, "", nil, &out, nil); err != nil {
+		return nil, fmt.Errorf("failed to list secrets: %w", err)
+	}
+
+	return out, nil
+}
+
+func (f *Client) CreateSecret(ctx context.Context, in fly.CreateSecretRequest) error {
```

Comment:
I was thinking more the other day about the shape of the API for import versus generating secrets. I think it's fine for these to be combined in the internal petsem API if that's easier, but I think it might be less confusing for users if we expose separate APIs for them to use for these two actions.

Reply:
done.